### DISCUSSION TITLE
Mark watched on rating

### DISCUSSION THREAD

**Discussion by cybermonk4** [trakt/showly#869] [Ideas / Feature Requests]
When an episode is rated, mark it as read. This seems logical because if you are rating something, then you should have watched it. A setting toggle for this would save rating then clicking that it was watched

**Comment by michaldrabik** [comment]
This feature is actually a premium feature for Play Store users called "Quick Rate". Whenever you mark an episode or a movie as watched you are then presented rating window immediately.

**Reply by cybermonk4** [reply]
I am aware of this feature but it's use is limited to only if something is on your list and it's the current episode. Accessing a show in other screens, such as opening it and going to a specific season, there is no quick rating, even when marking an episode as watched from the list of episodes in the season. If quick rate could be extended to these screens as well, as in, ANYTIME you mark something as watched, rating would pop up, then it would essentially be this request.

**Comment by cybermonk4** [comment]
An additional thought, quick rating should include a skip button. There are times I watch something and want to log that yet not provide a rating.
